hail/python/hailtop/aiocloud/aiogoogle/client/storage_client.py

```diff
@@ -588,10 +599,37 @@ class GoogleStorageAsyncFS(AsyncFS):
 
     def __init__(self, *,
                  storage_client: Optional[GoogleStorageClient] = None,
+                 bucket_allow_list: Optional[List[str]] = None,
                  **kwargs):
         if not storage_client:
             storage_client = GoogleStorageClient(**kwargs)
         self._storage_client = storage_client
+        if bucket_allow_list is None:
+            bucket_allow_list = []
+        self.storage_location_cache = bucket_allow_list
```

Comment:
nit: might be good to encode the fact that these buckets have been "allowed" in the name instead of just `storage_location_cache`.

Reply:
fixed